node-runtime/src/http-server.ts

```diff
@@ -379,7 +379,7 @@ export class SdkgenHttpServer<ExtraContextT = unknown> {
               const argName = ann.bodyVariable;
               const arg = op.args.find(x => x.name === argName);
 
-              if (req.headers["content-type"] === "application/json") {
+              if (req.headers["content-type"]?.match(/application\/json/iu)) {
```

Comment:
I personally prefer RegExp.test() which seems proper for an "if"

```suggestion
              if (/application\/json/iu.test(req.headers["content-type"])) {
```

But consider this change as optional